Jenkinsfile

```diff
@@ -0,0 +1,35 @@
+elifePipeline {
+    def commit
+    DockerImage image
+    stage 'Checkout', {
+        checkout scm
+        commit = elifeGitRevision()
+    }
+
+    node('containers-jenkins-plugin') {
+        stage 'Build images', {
+            checkout scm
+            dockerComposeBuild(commit)
+        }
+
+        stage 'Project tests', {
+            try {
+                sh 'docker-compose run --rm --entrypoint 'sh -c' csv-generator ./project_tests.sh'
+                sh 'docker-compose run --rm db-manager ./project_tests.sh'
```

Comment:
How would we run the tests in parallel? (Although that might lead to a confusing output?)

BTW db-manager doesn't have tests in this version but it has in another PR